### PR TITLE
Mailing list - UI, backend, and CSV export.

### PR DESCRIPTION
**Pull request by nein09**
This is a big one. I suggest that we go over it together @eldang so that I can have another look at it with some fresh eyes. But you're also welcome to look at it and see if you notice anything fishy before that.

### CODE REVIEW

File: disasterinfosite/static/css/app.css [hunk 1]
```diff
@@ -571,9 +641,3 @@ label {
   }
 }
 
-.smart911-logo {
```
Comment (nein09) [inline disasterinfosite/static/css/app.css:581]: I guess I moved that? I'm not sure why, but it's on line 418 now.

File: disasterinfosite/admin.py [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 ######################################################
 # GENERATED CODE GOES HERE
 # DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
 # adminModelImports
 from .models import EmbedSnugget, TextSnugget, SnuggetSection, SnuggetSubSection, Location, SiteSettings, SupplyKit, ImportantLink, Fire_Burn_Probability2, Flood_Worst_Case, EQ_Fault_Buffer, Fire_Worst_Case_ph2, EQ_Fault_Worst, EQ_Historic_Distance, EQ_Fault_Shaking, Flood_FEMA_DFRIM_2015, Fire_Hist_Bound, winterstorm, summerstorm, Landslide_placeholder2, Flood_Channel_Migration_Zones
 # END OF GENERATED CODE BLOCK
 ######################################################
-from .models import PastEventsPhoto, DataOverviewImage
+from .models import PastEventsPhoto, DataOverviewImage, UserProfile
+from.actions import export_as_csv_action
```
Comment (eldang) [inline disasterinfosite/admin.py]: does there need to be a space before `.actions` ?

File: disasterinfosite/static/js/app.js [hunk 3]
```diff
@@ -138,7 +138,7 @@ $( document ).ready(function() {
 
   function submitLocation(lat,lng) {
     // reload the page with the lat,lng
-    document.location =  encodeURI(document.location.hash + "?lat=" + lat + "&lng=" + lng + "&loc=" + location_query_text);
+    document.location =  encodeURI('/' + "?lat=" + lat + "&lng=" + lng + "&loc=" + location_query_text);
```
Comment (eldang) [inline disasterinfosite/static/js/app.js]: When the site is deployed in a subdirectory, the leading `/` pops us out of that subdirectory, breaking the link.

File: disasterinfosite/templates/user_interactions.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        <a href="#user-login-container" class="button radius caps button--login">Log In</a>
+        <a href="#user-signup-container" class="button radius caps button--signup">Sign Up</a>
+      </div>
+    </div>
+  <div id="user-login-container" class="row vert-pad-medium">
+    <form id="user-login__form">
+      {% csrf_token %}
+      <div class="small-12 medium-6 columns">
+        <label>Username (your email address)
+          <input type="email" id="user-login__username" required placeholder="Required."/>
```
Comment (eldang) [inline disasterinfosite/templates/user_interactions.html]: Currently it is possible for admin users' usernames to not be email addresses.  We should either change that or loosen the validation here.

**Pull request merged**